app/assets/stylesheets/application.scss

```diff
@@ -1,14 +1,14 @@
-/*
- *= require jquery.ui.autocomplete
- *= require vendor
- *= require scoped_search
- *= require multi-select
- *= require select2
- *= require select2-bootstrap
- *= require_self
- *= require_tree .
- */
-
+@import "jquery.ui.theme";
+@import "jquery.ui.core";
+@import "jquery.ui.menu";
```

Comment:
do we need all of the above just for the autocomplete function? if we do, worth mentioning it, if not, i guess they can be removed?